components/cloud_start_trial/request_business_email_modal.tsx

```diff
@@ -57,12 +60,15 @@ const RequestBusinessEmailModal = (
     const handleEmailValues = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
         const email = e.target.value;
         setEmail(email.trim().toLowerCase());
+
         validateEmail(email);
     }, []);
 
-    const validateEmail = useCallback(async (email: string) => {
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Nit: I don't think lint disable is needed. I don't see complaints in my editor or in `npm run check` if I remove locally. Do you? From what I can tell, every closed over variable value is from a function that has a stable identity. If there is some dep not passed in, useCallback + debounce + missing deps leads to hard to find bugs.